serverless-functions/src/functions/features/callback-and-voicemail/studio/wait-experience.protected.js

```diff
@@ -154,12 +155,13 @@ exports.handler = async (context, event, callback) => {
     holdMusicUrl = domain + holdMusicUrl;
   }
 
-  const { Digits, CallSid, enqueuedWorkflowSid, mode, enqueuedTaskSid, skipGreeting } = event;
+  const { Digits, CallSid, QueueSid, mode, enqueuedTaskSid, skipGreeting } = event;
 
   switch (mode) {
     case 'initialize':
       // Initial logic to find the associated task for the call, and propagate it through to the rest of the TwiML execution
       // If the lookup fails to find the task, the remaining TwiML logic will not offer any callback or voicemail options.
+      const enqueuedWorkflowSid = await VoiceOperations.fetchVoiceQueue({ context, queueSid: QueueSid }).friendlyName;
```

Comment:
A few things here:

- This function needs an `attempts: 0` param passed into it so the retry handler works properly. (Side note, we should probably fix this requirement separately)
- We need to wrap the function call and the await in parentheses so that the friendlyName property isnt accessed until the promise resolves (currently this returns undefined as a result)
- afaict, fetchVoiceQueue's return object has a queueProperties member that holds friendlyName, not on the top level of the return object (currently this returns undefined as a result)

Here's how I got this working:
```
const enqueuedWorkflowSid = (await VoiceOperations.fetchVoiceQueue({ context, queueSid: QueueSid, attempts: 0 }))
        .queueProperties.friendlyName;
```